### DISCUSSION TITLE
Tailwind ignoring custom colors, but only in some cases

### DISCUSSION THREAD

**Discussion by jchaselubitz** [tailwindlabs/tailwindcss#6798] [Help]
I just upgraded to version "3.0.8" and updated plugins accordingly. My project is Nextjs 12. 
MacOS: 12.1
Brave: Version 1.33.106 Chromium: 96.0.4664.110 (Official Build) (arm64)

Problem: I have custom colors defined in tailwind.config.js. I use these colors throughout the site. They work on text, but they don't appear to work on either `bg` or `border` classes. They DO however work when I apply a modifier, such as `hover`. 

For example, `cLightBlue: '#275A8F'` works on the following: 
`text-cLightBlue `
`hover:bg-cLightBlue`

it does NOT work (is ignored) on the following:
`bg-cLightBlue`
`border-cLightBlue` 

This is only the case with custom colors. Built in colors work fine.

**Comment by adamwathan** [comment]
Hey! Can you share a reproduction?

**Reply by jchaselubitz** [reply]
It seems to work fine when I try it in the Tailwind Playground, but I am including a link to that demonstration so that you can see my configuration: https://play.tailwindcss.com/bumL7E20Eh

I have taken a screen recording of the problem running on my own system. First I show that the custom color, `cLightBlue`, works fine on text. Then I show that it does not work on the background of a button, that it does work when the hover modifier is applied, and that a non-custom color also works.
https://www.youtube.com/watch?v=kY5MpHL4FnY

(I realize this isn't the most helpful reproduction, but I am not sure how else to demonstrate it. Please let me know if there is something else you need to see.)

Another note: I am running this in the latest version of Docker
